linetools/analysis/zlimits.py

```diff
@@ -23,7 +24,8 @@ class LineLimits(object):
     zlim : tuple of floats
       Redshift limits for a line
       Defined as wave/wrest - 1.
-    wvlim : Quantity array
+      or
+    wvlim : Quantity array, only if _wrest is set
       wavelength limits for the line in observer frame
```

Comment:
this docstring `only if _wrest is set` is confusing. Maybe say that this is an optional parameter that applies for zLimits to individual AbsLines (for which a single wrest makes sense)?

Reply:
improved